Code/GraphMol/FileParsers/MolFileParser.cpp

```diff
@@ -2211,6 +2211,12 @@ void ParseV3000AtomProps(RWMol *mol, Atom *&atom, typename T::iterator &token,
                          const T &tokens, unsigned int &line,
                          bool strictParsing) {
   PRECONDITION(mol, "bad molecule");
+  // Check if atom is initialized before proceeding
```

Comment:
This won't end up actually changing anything.
The next line down (the PRECONDITION) will also throw an exception if atom is null.

Reply:
Hii @greglandrum Thanks for reviewing i had done some update in code can you please review again

Reply:
@Shivam7-1 the changes you suggest here do not seem to be connected to the oss-fuzz problem you reference in the PR comment. That one seems to be arising from line 1254 of MolFileParser.cpp and have no connection to the setting of properties. Why do you think this fix is relevant?

Another comment on these changes: I believe that they should be unnecessary: everywhere in the code that uses one of the properties you add uses `getPropIfPresent()`, which works perfectly ok if the referenced property is not there.